features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/services/login/login_callback.jag

```diff
@@ -68,7 +73,7 @@
     var cookie = {
         'name': 'AM_ACC_TOKEN_DEFAULT_P2',
         'value': accessTokenPart2,
-        'path': site.context + "/",
+        'path': app.context + "/",
```

Comment:
We shouldn't need to rewrite cookies. The proxy server can do it on behalf of us: Ex:

proxy_cookie_path /publisher /custom-publisher;

Reply:
As offline discussed  with @malinthaprasan , we will address this change later.